cmake/projects/aws_lambda_cpp/hunter.cmake

```diff
@@ -0,0 +1,17 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME aws_lambda_cpp
+    VERSION "0.2.7"
```

Comment:
Since you patched it, the version number should be `v0.2.7-p0` and the branch name should be `hunter-0.2.7`. I renamed the branch and made a new release for you here: https://github.com/cpp-pm/aws-lambda-cpp/releases/tag/v0.2.7-p0

Reply:
Thanks for doing that. Are you also able to remove the `hunter-v0.2.7` branch? It seems that as a protected branch, the renaming failed.